recipes/pysftp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pysftp" %}
+{% set version = "0.2.9" %}
+{% set sha256 = "fbf55a802e74d663673400acd92d5373c1c7ee94d765b428d9f977567ac4854a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 2
```

Comment:
Please keep the build `number` at `0`. You can increment it in the feedstock if you make a change that doesn't involve a change in version number of the package. No need to increment here.